.circleci/scripts/build-execution-plan.mjs

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env zx
+
+const outputPath = "/tmp/parameters.json";
+
+const isOnMasterOrSupportBranch = process.env.CIRCLE_BRANCH === "master" || process.env.CIRCLE_BRANCH.match(/\d+\.\d+\.x/);
+if (isOnMasterOrSupportBranch) {
+    await fs.outputJson(outputPath, {});
+}
+
+let circleCiPR = process.env.CIRCLE_PULL_REQUEST;
+
+if (!circleCiPR) {
+    if (process.env.CIRCLECI) {
+        $`circleci step halt`;
+    }
+
+    process.exit(0);
+}
+
+const cleanedPRId = circleCiPR.substring(circleCiPR.lastIndexOf("/") + 1);
+
+const url = `https://api.github.com/repos/gravitee-io/gravitee-api-management/pulls/${cleanedPRId}`;
+
+const response = await fetch(url);
+if (!response.ok) {
+    // TODO: log error?
+    console.log(await response.text());
+}
+
+const body = await response.json();
+const targetBranch = body.base.ref;
+
+console.log(`Target branch is ${targetBranch}`);
+
+const modifiedFilesAsString = await $`git diff --name-only origin/${targetBranch}`;
```

Comment:
Juste 📝  
I was wondering if we could use yarn for monorepo and to make this kind of diff with https://yarnpkg.com/cli/workspaces/list and `--since master` options

It could also allow to add commit-lint ext... to global for monorepo